core/templates/pages/learner-dashboard-page/new-learner-dashboard/community-lessons-tab.component.html

```diff
@@ -0,0 +1,654 @@
+      </p>
+      <div class="summary-container">
+        <div *ngFor="let tile of displayIncompleteLessonsList"
+             class="oppia-summary-tile-container"
+             (mouseenter)="tile.toggle = !tile.toggle"
+             (mouseleave)="tile.toggle = !tile.toggle">
+          <div *ngIf="getTileType(tile) === 'exploration'">
+            <i class="remove-icon fa fa-times"
+               *ngIf="tile.toggle || checkMobileView()"
+               aria-hidden="true"
+               (click)="openRemoveActivityModal('I18N_LEARNER_DASHBOARD_INCOMPLETE_SECTION', 'I18N_DASHBOARD_EXPLORATIONS', tile)"
+               ngbTooltip="Remove"
+               container="body"
+               placement="top">
+            </i>
+            <oppia-exploration-summary-tile [getExplorationId]="tile.id"
+                                            [getExplorationTitle]="tile.title"
+                                            [getLastUpdatedMsec]="tile.lastUpdatedMsec"
+                                            [getObjective]="tile.objective"
+                                            [getCategory]="tile.category"
+                                            [getRatings]="tile.ratings"
+                                            [getNumViews]="tile.numViews"
+                                            [getThumbnailIconUrl]="tile.thumbnailIconUrl"
+                                            [getThumbnailBgColor]="tile.thumbnailBgColor">
+            </oppia-exploration-summary-tile>
+          </div>
+          <div *ngIf="getTileType(tile) === 'collection'">
+            <i class="remove-icon fa fa-times"
+               *ngIf="tile.toggle || checkMobileView()"
+               aria-hidden="true"
+               (click)="openRemoveActivityModal('I18N_LEARNER_DASHBOARD_INCOMPLETE_SECTION', 'I18N_DASHBOARD_COLLECTIONS', tile)"
+               ngbTooltip="Remove"
+               container="body"
+               placement="top">
+            </i>
+            <oppia-collection-summary-tile [getCollectionId]="tile.id"
+                                           [getCollectionTitle]="tile.title"
+                                           [getLastUpdatedMsec]="tile.lastUpdatedMsec"
+                                           [getObjective]="tile.objective"
+                                           [getNodeCount]="tile.nodeCount"
+                                           [getCategory]="tile.category"
+                                           [getThumbnailIconUrl]="tile.thumbnailIconUrl"
+                                           [getThumbnailBgColor]="tile.thumbnailBgColor">
+            </oppia-collection-summary-tile>
+          </div>
+        </div>
+        <div class="oppia-toggle-button-section" *ngIf="totalIncompleteLessonsList.length > 3">
+          <button class="btn oppia-toggle-btn"
+                  (click)="handleShowMore('incomplete')"
+                  *ngIf="!showMore['incomplete']">
+                  Show More
```

Comment:
For 'Show More', do you think it should just show everything else or show it in steps of 6 or something? So at each step, there will be both 'Show More' and 'Show Less' shown.

Reply:
It would be tedious for the user to click show more many times when they want to search for a particular card. It would be simpler to show all the cards when they click 'show more' and hide all the cards when they click 'show less'. What do you think?